paddle/fluid/operators/controlflow/while_op_helper.cc

```diff
@@ -232,5 +232,16 @@ bool GetCondData(const framework::LoDTensor &cond) {
   return cpu_cond->data<bool>()[0];
 }
 
+bool FindVarName(const std::string &name,
```

Comment:
FindVarName is not a good name. C++ "find" usually returns something such as the std::find you used below. Please rename it such as "StrInVaraiableNameMap".

Reply:
Thanks, done.